src/drawing/__tests__/DrawingAnnotations-test.tsx

```diff
@@ -232,4 +232,28 @@ describe('DrawingAnnotations', () => {
             },
         );
     });
+
+    describe('state changes', () => {
+        test.each`
+            drawn         | initialDrawn | numberOfCalls
```

Comment:
This is a little hard to read. Can we put the "base" state on the left, followed by the change, followed by the expected result? Can we name them as such?

Reply:
Yeah I'll rename / reorder this test